GPU/Common/DrawEngineCommon.h

```diff
@@ -30,10 +30,14 @@
 
 class VertexDecoder;
 
+// int maxVerts = DECODED_VERTEX_BUFFER_SIZE / 2 / vertexSize; =~ 54000
+// 65536 * 6 * 2
+
+
 enum {
 	VERTEX_BUFFER_MAX = 65536,
-	DECODED_VERTEX_BUFFER_SIZE = VERTEX_BUFFER_MAX * 64,
-	DECODED_INDEX_BUFFER_SIZE = VERTEX_BUFFER_MAX * 16,
+	DECODED_VERTEX_BUFFER_SIZE = VERTEX_BUFFER_MAX * 2 * 36,  // 36 == sizeof(SimpleVertex)
+	DECODED_INDEX_BUFFER_SIZE = VERTEX_BUFFER_MAX * 6 * 6 * 2,   // * 6 for spline tessellation, then * 6 again for converting into points/lines, and * 2 for 2 bytes per index
```

Comment:
So 4.5MB each, from previously 4MB + 1MB.  I think it's worth the increase in memory of 4MB more.  Most will hopefully not be committed anyway.

-[Unknown]